source/mesh_deformation/fastscapecc.cc

```diff
@@ -0,0 +1,698 @@
+#include <iostream>
+
+#include <xtensor/xarray.hpp>
+#include <xtensor/xmath.hpp>
+#include <xtensor/xview.hpp>
+#include <xtensor/xadapt.hpp>
+
+#include <fastscapelib/flow/flow_graph.hpp>
+#include <fastscapelib/flow/sink_resolver.hpp>
+#include <fastscapelib/flow/flow_router.hpp>
+#include <fastscapelib/grid/raster_grid.hpp>
+#include <fastscapelib/eroders/diffusion_adi.hpp>
+#include <fastscapelib/eroders/spl.hpp>
```

Comment:
We're going to have to figure out a way to disable compilation of this file when fastscape wasn't found during configuration. The way this is typically done is by `#include`ing `global.h` and then having a `#ifdef ASPECT_WITH_FASTSCAPE_CC` or some such.